app/models/card.rb

```diff
@@ -1,4 +1,9 @@
 class Card < ActiveRecord::Base
+  # scope :unviewed, -> {find_by_sql "SELECT * FROM cards WHERE review_date <= '2015-06-12'"} 
+  scope :unviewed, -> {find_by_sql "SELECT * FROM cards WHERE review_date <= (CURRENT_DATE - integer '3') ORDER BY random()"} 
```

Comment:
Line is too long. [126/80]<br>Space missing inside {.<br>Space missing inside }.<br>Trailing whitespace detected.